common/src/java/com/zimbra/common/gql/GqlConstants.java

```diff
@@ -52,4 +52,116 @@
 
     // end session constants
     public static final String CLEAR_COOKIES= "clearCookies";
+
+    // search constants
+    public static final String SEARCH_PARAMS = "params";
+    public static final String SEARCH_REQUEST = "SearchRequest";
```

Comment:
We will want to prefix the classes with CLASS for clarity like in cmac's PR. Can merge for now I will go back and update all these.